components/footer/FooterList2.js

```diff
@@ -0,0 +1,22 @@
+export const initiativeLinks = [
```

Comment:
Rename the file to more meaningful name for this array. You can also define this array in `FooterList.js`.